core/src/cubos/core/gl/palette.cpp

```diff
@@ -1,5 +1,8 @@
 #include <cubos/core/gl/palette.hpp>
 
+#include <cubos/core/log.hpp>
```

Comment:
ditto

Reply:
Before it was being included in another header, but with the refactor now we have to include it directly.